### PR TITLE
Fix exit code status check

### PR DESCRIPTION
**Pull request by edgarasg**
fix exit code status check for nginx service reload "if" statement

### CODE REVIEW

**Comment by safeith** [comment]
I have it but did not restart nginx

**Comment by fevangelou** [comment]
Thanks @edgarasg but out of curiosity, how is your solution different to the existing implementation which is just more verbose?

**Comment by edgarasg** [comment]
`-eq` operator should be used to compare integers. But `==` works also :)

File: nginx/utilities/https_vhosts.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ COUNTER="0"
 function generate_https_vhosts {
 	if [ -f /etc/nginx/utilities/https_vhosts.php ]; then
 		RUN_CHECK=$(/usr/bin/php -c /dev/null /etc/nginx/utilities/https_vhosts.php)
-		if [[ $RUN_CHECK == 1 ]]; then
+		if [[ $? -eq 1 ]]; then
 			service nginx reload;
```
Comment (safeith) [inline nginx/utilities/https_vhosts.sh]: /bin/service nginx restart install better

**Comment by ahlul** [comment]
This fix works for me :) .. with default script it never reload nginx (centos 7) when default_https.conf is updated.

**Comment by edgarasg** [comment]
Happy to hear!

**Comment by fevangelou** [comment]
@ahlul The problem is probably a broken /etc/crontab file because of wrong line endings. Open your /etc/crontab file via the Engintron WHM app and just hit save.

**Pull request closed without merging**